torchgeo/datasets/spacenet.py

```diff
@@ -93,7 +93,10 @@ def __init__(
             RuntimeError: if ``download=False`` but dataset is missing
         """
         self.root = root
-        self.image = image  # For testing
+        if image:
+            self.image = image
+        else:
+            self.image = list(self.imagery.keys())[0]
```

Comment:
@ashnair1 had to hack your SpaceNet datasets a bit. `root` can't be optional unless all things after it are optional, including `image`.